explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts

```diff
@@ -304,9 +305,12 @@ export const buildDescription = (
 export const buildDetails = (
   projectsResponse: ProjectsResponse
 ): React.ComponentProps<typeof C.Details> => {
-  const project = getProjectResponse(projectsResponse);
+  const keyValuePairs = new Map<Key, Value>();
+  for (const [key, value] of mapProjectDataSummary(projectsResponse)) {
+    keyValuePairs.set(DATA_SUMMARY_DISPLAY_TEXT[key], value);
```

Comment:
Converting from `DATA_SUMMARY` TO `DATA_SUMMARY_DISPLAY_TEXT` one line `310` will give us the opportunity to render a tooltip, should we wish to, in the future.